src/documentation/Usage/ResourcesDesigners.stories.mdx

```diff
@@ -16,54 +15,63 @@ import { iconAddGlyph } from '@wfp/icons';
   }}
 />
 
```

Comment:
You can also use `markdown` instead. For example a blockquote could look like this

```
> Title
>
> Content
```

Reply:
Thanks @Utzel-Butzel, I have actually tried to use it, also for the "Help us improve" section at the bottom of the page, but for some reasons it was not rendering in my local env. Should I change some settings?

Reply:
Can you share the source?

Reply:
Found out why. There was no space between the elements and the markdown was not rendered properly, pushed the changes now. For what concerns the blockquote, I think the usage of semibold and bold does not help the contrast between the title and the content, as the last stands out a lot more than the title itself, we may want to review it.